src/components/SignUp/index.scss

```diff
@@ -0,0 +1,13 @@
+.signup-wrapper{
```

Comment:
It's better to put this CSS scripts in a file named _index.scss_ 
Just to organize the CSS better.
`/home/kmishra/webD/teesco-website/src/pages/SignUpPage`